language/backend/en-GB/en-GB.com_j2store.ini

```diff
@@ -118,16 +118,20 @@ J2STORE_ADVANCED_SETTINGS="Advanced Settings"
 J2STORE_INTEGRITY_CHECK="Integrity Check"
 J2STORE_CONGRATULATIONS="Congratulations!"
 J2STORE_POSTCONFIG_WELCOME_MESSAGE="You are just a few steps away from building your online shop with J2Store"
-J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br> These can be managed at J2Store -> Configuration as well."
+J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br> These can be managed anytime at J2Store -> Configuration."
+J2STORE_POSTCONFIG_WHATTHIS_LINKTEXT="J2Store -> Configuration"
 J2STORE_INTEGRITY_CACHE_PLUGIN_ENABLED="System - Cache plugin is ENABLED. Recommended Setting: DISABLED"
+J2STORE_INTEGRITY_CACHE_PLUGIN_LABEL="System - Cache Plugin"
+J2STORE_INTEGRITY_SYSTEM_PLUGIN_LABEL="System - J2Store Plugin"
 J2STORE_INTEGRITY_SYSTEM_PLUGIN_ENABLED="System - J2Store plugin is ENABLED."
+
 J2STORE_POSTCONFIG_STORE_SETUP_DONE_ALREADY="First run wizard was already executed. If you need to alter the settings, please go to J2Store -> Configuration"
 
 J2STORE_DEFAULT_TAX_RATE="Default Tax Rate (optional)"
 J2STORE_DEFAULT_TAX_RATE_DESC="You can choose to create a default tax rate. You can also do this after completing this wizard. This is purely optional"
-J2STORE_POSTCONFIG_LBL_MANDATORYINFO="Mandatory Information"
+J2STORE_POSTCONFIG_LBL_MANDATORYINFO="Free Version License / Support"
 J2STORE_AGREE_SET_UP_STORE="Agree & Setup Store"
-J2STORE_POSTCONFIG_LBL_ACCEPTLICENSE="I have read, understood and accept the license of the software"
+J2STORE_POSTCONFIG_LBL_ACCEPTLICENSE="I have read, understand, and accept the license of the software."
```

Comment:
this should be reverted - it was correct english

Reply:
@brianteeman the verbs read, understand, and accept should all be in the present (or past) tense to maintain parallel structure. the previous language string is using a mix of both past and present tense. I don't care either which way we go, but we should be consistent.
The other alternative could be  I have read, understood, and accepted the license of the software?

Reply:
i have **read** the license, I have **understood** the licence and by clicking here I **accept** the licence

Reply:
even better.